examples/logger/package.yaml

```diff
@@ -0,0 +1,62 @@
+name:                logger-example
+version:             0.1.0.0
+github:              "stackbuilders/atomic-write"
+license:             BSD-3-Clause
+author:              "@BarbDMC"
+maintainer:          "amejia@stackbuilders.com"
+copyright:           "2023 @BarbDMC"
+
+extra-source-files:
+- README.md
+- CHANGELOG.md
+
+# Metadata used when publishing your package
+# synopsis:            Short description of your package
+# category:            Web
+
+# To avoid duplicated efforts in documentation and dealing with the
+# complications of embedding Haddock markup inside cabal files, it is
+# common to point users to the README.md file.
+description:         Please see the README on GitHub at <https://github.com/githubuser/logger-example#readme>
+
+dependencies:
+- base >= 4.7 && < 5
+- atomic-write >= 0.2
+- text >= 2.0
+- filepath >= 1.4 && < 2
+
+ghc-options:
+- -Wall
+- -Wcompat
+- -Widentities
+- -Wincomplete-record-updates
+- -Wincomplete-uni-patterns
+- -Wmissing-export-lists
+- -Wmissing-home-modules
+- -Wpartial-fields
+- -Wredundant-constraints
+
+library:
+  source-dirs: src
+
+executables:
+  logger-example-exe:
+    main:                Main.hs
+    source-dirs:         app
+    ghc-options:
+    - -threaded
+    - -rtsopts
+    - -with-rtsopts=-N
+    dependencies:
+    - logger-example
+
+tests:
```

Comment:
I do not think we need this as there are no tests

Reply:
You're right, I removed those test dependencies.